pandas/tests/arithmetic/test_array_ops.py

```diff
@@ -34,3 +35,15 @@ def test_object_comparison_2d():
     right.flags.writeable = False
     result = comparison_op(left, right, operator.ne)
     tm.assert_numpy_array_equal(result, ~expected)
+
+
+@pytest.mark.parametrize(
+    "data, expected", [([0, 1, 2], [0, 2, 4])],
+)
+def test_extension_array_add(box_1d_array, data, expected):
+    # GH22606 Verify operators with Extension Array and list-likes
+    ser = Series(data, dtype="Int64")
+    results = ser + box_1d_array(data)
+
```

Comment:
can you test reversed args as well, eg. box_1d_array(data) + ser

Reply:
Done